### DISCUSSION TITLE
WebAuthN Error while registering

### DISCUSSION THREAD

**Discussion by belerovon** [dani-garcia/vaultwarden#4097] [Q&A]
When using Keycloak and want register a new Security-Token - Vaultwarden jump in and following error occured:
TypeError: undefined is not an object (evaluating '_b.authenticatorSelection.authenticatorAttachment')

When Browser extension is disabled, I can use my Yubikey.

Tested with Safari, Chrome, Firefox - behavior all the same.

Maybe this Issue is related with this one:
https://github.com/bitwarden/clients/issues/6769

**Comment by BlackDex** [comment]
I think this is a client issue. Nothing server side.
But to be sure, just check the `/admin/diagnostics` and see if all is ok. Other then that, I'm not sure Vaultwarden can do anything here.
